Tests/DatadogTests/Datadog/RUM/RUMEventOutputs/RUMEventFileOutputTests.swift

```diff
@@ -62,7 +64,9 @@ class RUMEventFileOutputTests: XCTestCase {
             jsonString: """
             {
                 "attribute": "foo",
-                "context.custom.attribute": "value"
+                "context": {
+                    "custom.attribute": "value"
+                }
```

Comment:
I guess this is fine?

Reply:
Totally 👍. For our backend it's the same.